eng/Build.props

```diff
@@ -115,9 +119,10 @@
         <NodeJsProjects Include="
                           $(RepoRoot)src\Components\Web.JS\Microsoft.AspNetCore.Components.Web.JS.npmproj;
                           $(RepoRoot)src\SignalR\**\*.npmproj;
-                          $(RepoRoot)src\Middleware\**\*.npmproj;
```

Comment:
Why this removal?

Reply:
Because there are no `npm` projects in the Middleware folder. I vaguely mentioned this in the description:

> remove reference to non-existent .npmproj files